articles/policies/rate-limits.md

```diff
@@ -77,115 +77,31 @@ The following rate limits apply:
 - For __non-production tenants__ of enterprise customers, usage of the Management API is restricted to 2 requests per second (and bursts up to 10 requests).
 - For __paid__ tenants, usage of the Management API is restricted to 15 requests per second (and bursts up to 50 requests).
 
-The aforementioned rate limits include calls made via [Rules](/rules).
+The aforementioned rate limits include calls made via [Rules](/rules) and via the Manage Dashboard.
```

Comment:
Not sure if we want to clarify this, but limits associated with the dashboard are always the ones for prod tenants.